### DISCUSSION TITLE
Where to find "official" documentation ?

### DISCUSSION THREAD

**Discussion by caring-coder** [iamrasel/lunar-launcher#71] [General]
This launcher is really interesting, I'm wondering if there is a dedicated place where you can find a kind of "user manual".
I'm thinking about a few pages or a Q&A document with step by step how-to setup weather API key or RSS feed, or even adding app to quick action windows. 
Right now, it is a game of trial and errors.

**Comment by iamrasel** [comment]
I am not good at writing documentations or manuals, so there's nothing like that except the [wiki](https://github.com/iamrasel/lunar-launcher/wiki) pages (these are also poorly written though).

You could get an OpenWeatherMap api just by registering a free account on their website.

For rss feed, paste your favorite feed link (e.g. https://mosfet.net/feed) in the specific input box in Settings. Sorry that it still doesn't support some websites.

To add an app to the quick access screen, follow these- tap and hold the app from the app drawer, you'll see six boxes there. Click on any box/place from there to select the position for the app in quick access screen. That's the whole process.

**Reply by kaitou1412** [reply]
can you show me where is this quick access screen?
because i already fill those 6 boxes with application i chose, but i can't find the quick access section that shows them.
thanks in advance

![scrcpy_N5JREUreN6](https://github.com/user-attachments/assets/4fc66121-5b6b-416d-a8a9-435546089ced)
